pkg/kando/location_delete.go

```diff
@@ -66,7 +67,7 @@ func runLocationDelete(cmd *cobra.Command) error {
 
 // kopiaLocationDelete deletes the kopia snapshot with given backupID
 func kopiaLocationDelete(ctx context.Context, backupID, path, password string) error {
-	return kopia.DeleteSnapshot(ctx, backupID, path, password)
+	return snapshot.DeleteSnapshot(ctx, backupID, path, password)
```

Comment:
We should rename the method to `Delete` which will make `snapshot.Delete()` more readable